spec/ParseUser.spec.js

```diff
@@ -24,11 +24,6 @@ describe('allowExpiredAuthDataToken option', () => {
     expect(Config.get(Parse.applicationId).allowExpiredAuthDataToken).toBe(false);
   });
 
-  it('should default false', async () => {
```

Comment:
This seems to be related to a different option, why was this removed?

Reply:
sorry, I read it wrong. thought since its already true in test configuration, testing it to default value would be wrong. but anyway, like i said i set default to true on test config. so, do you have any suggestions on how I can test to ensure the default is as expected?

Reply:
Well, first, the test you removed `should default false` is under describe `allowExpiredAuthDataToken option`, so this is a different, and I believe unrelated option, so this removal should be reverted. You ask how to test a default value - that is exactly what the test that you removed does, but for a different option. You can just copy that test and add it under the correct `describe` that is related to the `allowClientClassCreation` option.

Reply:
Yes i know, i will revert that but, as you know, with the change of `allowClientClassCreation` default to false, most of the test fails. so to avoid that, i set `allowClientClassCreation:true` on tests `defaultConfiguration` settings. You can see that on commits aswell. So, since its set to true on tests configuration, i cannot test it as exactly the same way with the one that i removed incorrectly. So its default false but configured to true. 

In order to test the `allowClientClassCreation:false` change, we either need to give `allowClientClassCreation:true` to the entire test with reconfigureServer to avoid fails (hundreds of tests will need to be changed), or we need to access to the default value in another way, which is exactly what I'm asking. How can we look at the `allowClientClassCreation`'s default value of the server?

Reply:
Did you try to call `reconfigureServer({ allowClientClassCreation: undefined });` to try to override the default config?

Reply:
@mtrezza while i writing a logic to get defaultvalues, i see that `allowClientClassCreation`'s default value is still true. i tested it via deleting `allowClientClassCreation:true` on tests defaultConfiguration. and to be sure, i set default value of `allowClientClassCreation` to false in alpha branch to see if this branch is broken somehow but setting it to false didnt work at alpha branch also.  And also i noticed that `allowExpiredAuthDataToken`'s default value test(which i removed incorrectly) also fails. which should be false according to definitions. I tested that on alpha branch and confirmed that something wrong with default values, maybe a merged pr broke that.

Reply:
@mtrezza sorry there was no failure on tests just something wrong on local. just made an update can you check